Edited the shell script to look for the main .jar file
under /usr/share/dirbuster/.
--- a/DirBuster-1.0-RC1.sh
+++ b/DirBuster-1.0-RC1.sh
@@ -1,3 +1,3 @@
 #!/bin/bash
-java -Xmx256M -jar DirBuster-1.0-RC1.jar
+java -Xmx256M -jar /usr/share/dirbuster/DirBuster-1.0-RC1.jar
 
